benten-core/src/main/java/com/intuit/benten/properties/BentenProxyConfig.java

```diff
@@ -56,6 +56,6 @@ public String getProtocol() {
     public void setProtocol(String protocol) {
         this.protocol = protocol;
     }
-    
```

Comment:
sorry for being picky but can you revert this change as there is no change except an extra space ?